Resources/AzureDevOps/JobTemplates/Build.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  AppImageName: ''
+  ContainerRegistryAdminUser: ''
+  ContainerRegistryPassword: ''
+
+jobs:
+- job: BuildNCSServiceTaxonomyEscoDbContainer
+  displayName: Build and publish NCS.ServiceTaxonomy.EscoDb container
+  pool: 'Hosted Ubuntu 1604'
+  steps:
+  - script: |
+      cd ./Resources/Dockerfiles
+      AppImageName=${{ parameters.AppImageName }}
+      # Lowercase the AppImage name to conform to Docker Image naming constraints
+      AppImageName=${AppImageName,,}
```

Comment:
it's definitely worth documenting what this does somewhere, as it's likely confuse anyone who's not channelling bash all the time ;)